UEBPrep/app/views/cards/_footer.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= social_share_button_tag("A CARD") %>
```

Comment:
Yes. Links me right to them with the card's URL ready to go. Well, other than it being a localhost address for now..